application/src/main/java/com/sanction/thunder/email/EmailConfiguration.java

```diff
@@ -28,4 +28,28 @@ public String getRegion() {
   public String getFromAddress() {
     return fromAddress;
   }
+
+  @NotEmpty
+  @JsonProperty("success-html")
```

Comment:
Can you make all of these `JsonProperty` use camel case instead of hyphens? This was a recent change.